cmd/createorupdate/createorupdate.go

```diff
@@ -26,10 +26,18 @@ import (
 
 var logLevel = flag.String("loglevel", "Debug", "valid values are Debug, Info, Warning, Error")
 
+var pc *plugin.Config
+
 // createOrUpdate simulates the RP
 func createOrUpdate(ctx context.Context, oc *v20180930preview.OpenShiftManagedCluster, entry *logrus.Entry) (*v20180930preview.OpenShiftManagedCluster, error) {
```

Comment:
Can you move plugin.Config to be passed as an argument in `createOrUpdate`? This is going to be useful once we share `createOrUpdate` between this code and the e2e framework.

Reply:
done